src/index.js

```diff
@@ -136,6 +136,17 @@ export const validateWhitelistOption = (whitelistOpt = []) => {
 
 let hasBeenLogged = false;
 
+const logPlugin = (plugin, targets, list) => {
+  const envList = list[plugin];
+  const filteredList = Object.keys(targets)
+  .reduce((a, b) => {
+    a[b] = envList[b];
+    return a;
+  }, {});
+  const logStr = `\n ${plugin} ${JSON.stringify(filteredList)}`;
```

Comment:
do we need the `\n` given its separate console.logs? I see that it's newlines? (Unless you think it looks better that way)


```
Using plugins:

 syntax-trailing-function-commas {}

Using polyfills:

 es6.typed.data-view {}

 es6.reflect.apply {}

 es6.reflect.own-keys {}

 es6.symbol.iterator {}

 es6.symbol.species {}

 es6.array.from {}
```

```
Using plugins:
syntax-trailing-function-commas {}

Using polyfills:
es6.typed.data-view {}
es6.reflect.apply {}
es6.reflect.own-keys {}
es6.symbol.iterator {}
es6.symbol.species {}
es6.array.from {}
```